docs/plugins_guidelines.md

```diff
@@ -0,0 +1,115 @@
+
+## Bevy Version Supported
+
+Indicating which version of your plugin works with which version of Bevy can be helpful for your users. Some of your users may be using an older version of Bevy for any number of reasons. You can help them find which version of your plugin they should use. This can be shown as a simple table in your readme with each version of Bevy and the corresponding compatible version of your plugin.
+
+|bevy|bevy_awesome_plugin|
+|---|---|
+|0.4|0.3|
+|0.3|0.1|
+
+## Bevy Features
+
+You should disable Bevy features that you don't use. This is because with Cargo, features are additive. Features that are enabled for Bevy in your plugin can't be disabled by someone using your plugin. You can find the list of features [here](cargo_features.md).
+```
+bevy = { version = "0.4", default-features = false, features = ["..."] }
+```
+
+## Master Branch Tracking
+
+If you intend to track Bevy's master branch, you can specify the latest commit you support in your cargo.toml file:
+```
+bevy = { version = "0.4", git = "https://github.com/bevyengine/bevy", rev="509b138e8fa3ea250393de40c33cc857c72134d3", default-features = false }
+```
+You can specify the dependency [both as a version and with git](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations), the version will be used if using the dependency from [crates.io](https://crates.io), the git dependency will be used otherwise.
+
+Bevy is evolving very fast. You can use one of these badges to communicate to your users how closely you intend to track Bevy's master branch.
+
+|<div style="width:100px">badge</div>|<div style="width:200px">description</div>|code|
+|-|-|-|
+|[![Bevy tracking](https://img.shields.io/badge/Bevy%20tracking-master-lightblue)](https://github.com/bevyengine/bevy/blob/master/docs/plugins_guidelines.md#master-branch-tracking)|I intend to track master as much as I can|`[![Bevy tracking](https://img.shields.io/badge/Bevy%20tracking-master-lightblue)](https://github.com/bevyengine/bevy/blob/master/docs/plugins_guidelines.md#master-branch-tracking)`|
+|[![Bevy tracking](https://img.shields.io/badge/Bevy%20tracking-PR%20welcome-lightblue)](https://github.com/bevyengine/bevy/blob/master/docs/plugins_guidelines.md#master-branch-tracking)|I welcome PRs that will update my plugin to the current Bevy master (or for new releases)|`[![Bevy tracking](https://img.shields.io/badge/Bevy%20tracking-PR%20welcome-lightblue)](https://github.com/bevyengine/bevy/blob/master/docs/plugins_guidelines.md#master-branch-tracking)`|
+|[![Bevy tracking](https://img.shields.io/badge/Bevy%20tracking-master-lightblue)](https://github.com/bevyengine/bevy/blob/master/docs/plugins_guidelines.md#master-released%20version-tracking)|I will only follow released Bevy versions|`[![Bevy tracking](https://img.shields.io/badge/Bevy%20tracking-master-lightblue)](https://github.com/bevyengine/bevy/blob/master/docs/plugins_guidelines.md#master-released%20version-tracking)`|
```

Comment:
The "example" badge for "release tracking" says master :)

Reply:
bad copy pasta, fixed...